src/button/button-icon.scss

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+/// Mixin for button icons
+/// @arg fillColor - color to fill in icon
+@mixin iui-button-icon($fillColor: t(iui-icons-color-actionable)) {
+  width: $iui-icons-default;
+  height: $iui-icons-default;
+  transition: fill $iui-speed-fast ease-out;
+
+  @include themed {
+    fill: $fillColor;
```

Comment:
So as I was reviewing the visuals, I realized that the icon color is always the same as text color. So I think we'll be able to reduce complexity a lot just by using `fill: currentColor`, provided the `color` is set on the parent (which it should). We won't need to override the fill in various places.

Reply:
Please check code if I understood you correctly. Made some changes.

Reply:
I just approved the failing tests in [`584909a`](https://github.com/iTwin/iTwinUI/pull/363/commits/584909a0d034de52be998d406b54bdb8615fe2d2). It is a slight change in opacity of icons, but I think it might actually have a better outcome